src/plugins/links/public/editor/open_link_editor_flyout.tsx

```diff
@@ -44,6 +46,12 @@ export async function openLinkEditorFlyout({
       // wait for close animation before unmounting
       setTimeout(() => {
         if (ref.current) ReactDOM.unmountComponentAtNode(ref.current);
+
+        // return focus to the main flyout div to align with a11y standards
+        const flyoutElement = document.getElementById(mainFlyoutId);
```

Comment:
This works well when I add and edit links. Can we include logic to also reset this focus to the modal container when I delete a link using the inline trashcan icon?

Reply:
Done in 11468acc478b7ce8ec3a8bc36f0925d29aa27a6e :tada: